pcap/pcap.go

```diff
@@ -33,11 +33,42 @@ package pcap
 // See http://upstream-tracker.org/versions/libpcap.html
 
 #ifndef PCAP_ERROR_TSTAMP_PRECISION_NOTSUP  // < v1.5
+#define PCAP_ERROR_TSTAMP_PRECISION_NOTSUP -12
 
 int pcap_set_immediate_mode(pcap_t *p, int mode) {
   return PCAP_ERROR;
 }
 
+#define PCAP_TSTAMP_PRECISION_MICRO	0
+#define PCAP_TSTAMP_PRECISION_NANO	1
+
+#ifdef WIN32
+pcap_t *pcap_hopen_offline_with_tstamp_precision(intptr_t osfd, u_int precision,
```

Comment:
it looks like right now precision is being ignored?  maybe add a comment in-code saying why?

(same for fopen)

Reply:
Ah yeah, thats for libpcap < 1.5 (see ifndef above) since before that version libpcap didn't support changing the resolution (it's fixed to micro). I'll add a comment.